src/internal/packager/helm/chart.go

```diff
@@ -115,12 +115,6 @@ func (h *Helm) InstallOrUpgradeChart() (types.ConnectStrings, string, error) {
 			return nil, "", fmt.Errorf("unable to upgrade chart after %d attempts", h.retries)
 		}
 
-		spinner.Updatef("Performing chart uninstall")
```

Comment:
May be useful to warn the user that we couldn't uninstall and that if they want things gone they should do a `zarf package remove`

Reply:
included a message to recommend using `zarf package remove` in the returned error